tools/llm_bench/llm_bench_utils/hook_forward_whisper.py

```diff
@@ -18,8 +18,7 @@ def get_time_list(self):
             if 'enc_token_time' in data:
                 first_token_latency += data['enc_token_time']
             if 'dec_token_time' in data:
-                first_token_latency += data['dec_token_time'][0]
-                self.tm_list.extend(copy.deepcopy(data['dec_token_time'][1:]))
+                assert not data['dec_token_time']
```

Comment:
Why we need to have assert here ? 
If there is no particular need for throw an exception, I would replace it with log.,warning

Reply:
that is, theoretically there could be a decoder here, but all the cases you tried didn’t contain it?

Reply:
I only tried the cases covered by tests